.gitignore

```
@@ -33,3 +33,5 @@ yarn-error.log*
 .env
 /src/__tests__/__setup__/data.json
 .idea
+
+venv/*
```

We generally keep an empty newline in all files https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline
```suggestion
venv/*

```